R/sidebar.R

```diff
@@ -112,16 +156,30 @@ layout_sidebar <- function(sidebar = sidebar(), ..., fillable = FALSE, fill = TR
     columns_collapse <- rev(columns_collapse)
   }
 
+  sidebar_max_height_mobile <-
+    if (is.null(sidebar$max_height_mobile)) {
+      if (!is.null(height)) {
+        "fit-content(50%)"
+      } else {
+        "fit-content(250px)"
+      }
+    } else {
+      sprintf("fit-content(%s)", sidebar$max_height_mobile)
+    }
```

Comment:
Have you tested this in various browsers? The browser support doesn't seem super great https://caniuse.com/?search=fit-content

Reply:
Yes, I tested locally in Firefox, Chrome and Safari, but we could do more testing.

We'd be using `fit-content()` in a grid context; here's the direct [browser support](https://caniuse.com/mdn-css_properties_grid-template-rows_fit-content) for that. The other browser support results conflate `fit-content` as an intrinsic size (similar to `max-content`) when used in other properties. AFAICT `fit-content()` has been a part of CSS grid for a while.